docs/fabric/MAS_fabric/mas_fabric.md

```diff
@@ -37,10 +45,9 @@ The `fabric` module operates in a single mode, facilitating data routing and com
 3. For tile-to-tile communication, it manages data exchange in all four cardinal directions (North, East, West, South) using arrays of signals (`in_` and `out_` signals) for request validity, transaction details, and ready signals.
 4. Local tile IDs are generated based on tile positions for routing.
 
-## 5. Configuration and Control
+## 5. different configuration
 
-**Configuration Registers:**
-The `fabric` module does not include explicit configuration registers. Configuration and control may be implemented at a higher system level.
+In future we will be able to cange the size of the fabric by changing the ROW and COL paramters, we will also be able to connect different tiles and not just the mini_core_tile.
 
 ## 6. Testing and Verification
 
```

Comment:
Also here -> please at least mention the coverage points , what needs to be tested. not how its tested.
(back pressure, full bandwidth, stress tests)

Reply:
done